metaserver/metaserver_test.go

```diff
@@ -0,0 +1,980 @@
+package metaserver
```

Comment:
Since this relies on setup.sh, maybe move this into integration.

Reply:
As long as we can run it from there, that sounds fine to me.